tests/functional/spec/features/last_viewed_guidance/last_viewed_guidance.spec.js

```diff
@@ -0,0 +1,51 @@
+const JwtHelper = require("../../../jwt_helper");
+
+const AddressConfirmationPage = require('../../../generated_pages/last_viewed_question_guidance/address-confirmation.page.js');
+const HouseholdInterstitialPage = require('../../../generated_pages/last_viewed_question_guidance/household-interstitial.page.js');
+const PrimaryPersonListCollectorPage = require('../../../generated_pages/last_viewed_question_guidance/primary-person-list-collector.page.js');
+
+describe('Last viewed question guidance', function () {
+
+  const responseId = JwtHelper.getRandomString(16);
```

Comment:
Can we define the extra params for `openQuestionnaire` in one place, something like:
```javascript
  const resumableLaunchParams = {
    responseId: JwtHelper.getRandomString(16),
    userId: "test_user"
  };
```